app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -0,0 +1,80 @@
+<% content_for :title, t('page_titles.application_guidance') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @recruitment_cycle_year < 2024 %>
```

Comment:
Should the new continuous applications content be displayed if the `continuous_applications` feature flag is on too? (For testing purposes it might be useful?)